Sockets/BackEndMonitoringSockets/WindowsUtils.cpp

```diff
@@ -0,0 +1,47 @@
+#include "stdafx.h"
+#include ""
```

Comment:
What is this include?

Reply:
It is really strange. I see #include "PlatformUtils.h". I don't know why git doesn't show it.